scripts/update-generator/main.go

```diff
@@ -93,14 +94,31 @@ func copyFileToDir(filePath, dir string, isExecutable bool) error {
 	return nil
 }
 
+func copyDirToDir(fromDir, toDir string) error {
+	targetPath := filepath.Join(toDir, filepath.Base(fromDir))
+	fmt.Printf("Copying %s -> %s\n", fromDir, targetPath)
+
+	err := fileutils.Mkdir(targetPath)
+	if err != nil {
+		return errs.Wrap(err, "Could not create target path %s", targetPath)
+	}
+
+	err = fileutils.CopyFiles(fromDir, targetPath)
+	if err != nil {
+		return errs.Wrap(err, "Could not copy directory %s -> %s", fromDir, targetPath)
+	}
+
+	return nil
+}
```

Comment:
I think you can drop this and use fileutils instead:

http://github.com/ActiveState/cli/blob/dd597af58d16cdac0f3cf0694338598543ec15eb/internal/fileutils/fileutils.go#L537-L537

Reply:
It's probably better to use that function. And in trying to use it I discovered that it would error out if there was a directory in the source path. However a lot of the logic of this function will remain as it's just creating the target path.